From 7ae95ad6b6f57fae2526e410bac75aa4039c5d93 Mon Sep 17 00:00:00 2001
From: Ben Westover <kwestover.kw@gmail.com>
Date: Tue, 20 Sep 2022 23:15:38 -0400
Subject: [PATCH] Don't install manpages to /usr/usr

`data_files` shouldn't have `usr/` in it; this causes the manpages to be installed to `/usr/usr/share/man/man1` instead of `/usr/share/man/man1`.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index a63be643..a756c8af 100755
--- a/setup.py
+++ b/setup.py
@@ -56,7 +56,7 @@ def read_module(filename):
         ("share/bumblebee-status/themes", glob.glob("themes/*.json")),
         ("share/bumblebee-status/themes/icons", glob.glob("themes/icons/*.json")),
         ("share/bumblebee-status/utility", glob.glob("bin/*")),
-        ("usr/share/man/man1", glob.glob("man/*.1")),
+        ("share/man/man1", glob.glob("man/*.1")),
     ],
     packages=find_packages(exclude=["tests", "tests.*"])
 )
